MapboxNavigation/CarPlayManager.swift

```diff
@@ -249,6 +265,13 @@ public class CarPlayManager: NSObject {
         carWindow?.isHidden = true
         let timestamp = Date().ISO8601
         sendCarPlayDisconnectEvent(timestamp)
+        
+        // Upon disconnecting the device from carplay, the idle timer ideally should be reset to its initial state.
+        if let shouldDisableIdleTimer = delegate?.carplayManagerShouldDisableIdleTimer?(self) {
+            UIApplication.shared.isIdleTimerDisabled = !shouldDisableIdleTimer
+        } else {
+            UIApplication.shared.isIdleTimerDisabled = true
```

Comment:
should be false

Reply:
yeah I agree, this doesn't seem quite right, but it could either be the inverted boolean or the reuse of the flag with a different semantic meaning. 

Did you two resolve this during your pairing?